packages/jellyfish-network/src/Network.ts

```diff
@@ -4,9 +4,9 @@
 export type NetworkName = Network['name']
 
 /**
- * Network specific DeFi configuration.
- * They can be found in DeFiCh/ain project in file chainparams.cpp, under base58Prefixes
- */
+  * Network specific DeFi configuration.
+  * They can be found in DeFiCh/ain project in file chainparams.cpp, under base58Prefixes
+  */
```

Comment:
```suggestion
 * Network specific DeFi configuration.
 * They can be found in DeFiCh/ain project in file chainparams.cpp, under base58Prefixes
 */
```

You need to revert these documentation indentation changes. It's not following the correct default expected indentations defined here https://jsdoc.app/about-getting-started.html